packages/core/deploy/001_deploy_finder.js

```diff
@@ -0,0 +1,14 @@
+const func = async function(hre) {
```

Comment:
i'm guessing we'll add to these when we add contracts to the setup (like financial contracts, collateral whitelists, etc)?

Reply:
yup! For each contract we add extra functionality to run if the contract gets newly deployed.